variables.tf

```diff
@@ -321,3 +321,9 @@ variable "sysdig_agent_version" {
   description = "Optionally override the default Sysdig agent version. If the value is null, this version is set to the version of 'sysdig_agent_version' variable in the Observability agents module. To list available versions, run: `ibmcloud cr images --restrict ext/sysdig/agent`."
   default     = null
 }
+
+variable "sysdig_agent_tags" {
+  type        = list(string)
+  description = "List of tags to associate with the sysdig instance"
```

Comment:
This module deploys the sysdig agent, not instance

Reply:
Similarly, for logdna_agent_tags description will be changed.